tasks/main.yml

```diff
@@ -1,5 +1,12 @@
 # role tasks
 
+- name: enable and start firewalld
+  become: "{{ firewalld_become }}"
+  service:
+    name: firewalld
+    state: started
+    enabled: true
+
```

Comment:
I assume that firewalld never needs to be restarted with new rules? We don't have code for that anywhere in here.

Reply:
That's my understanding, yes. firewalld needs to be started to be able to apply rules (and *normally is* started by default on RHEL hosts), but does not need to be restarted for rule changes to take effect.